services/algolia-sync/src/components/graphql/query.js

```diff
@@ -0,0 +1,22 @@
+const createClient = require('./create-client');
+require('dotenv').config();
+
+const queryFromBase = async (query, tenantKey, input) => {
+  const context = {
+    headers: {
+      'x-tenant-key': tenantKey,
+      'x-cdn-image-hostname': 'base.imgix.net',
```

Comment:
This header was deprecated starting in `v1.x` of the monorepo -- the CDN hostname should be set on the primary site of the content. You shouldn't need to pass it through with the query.

Reply:
See the `sitemaps` service for more examples of this!

Reply:
Also, you should be able to pass the tenant key through to the apollo client by using the HttpLink. It looks like the only purpose of this file is to return a single method that injects the headers. See an example [here](https://github.com/base-cms/company-update/blob/master/services/graphql/src/schema/base.js#L13).

Reply:
> This header was deprecated starting in `v1.x` of the monorepo -- the CDN hostname should be set on the primary site of the content. You shouldn't need to pass it through with the query.

I removed the x-cdn